pkg/api/domains/domains.go

```diff
@@ -53,17 +55,23 @@ func NewClient(c *http.Client, url string, token string) *Client {
 
 func (c *Client) Get(ctx context.Context, id string) (DomainResponse, error) {
 	req := c.apiClient.DomainsApi.GetDomain(ctx, id)
+	logger.Debug("request", zap.Any("request", req))
 	res, httpResp, err := req.Execute()
+	logger.Debug("response", zap.Any("response struct", res), zap.Any("response http", httpResp), zap.Error(err))
 	if err != nil {
+		logger.Error("Get request.Execute return error", zap.Error(err))
```

Comment:
I think we do not need to name the actual function, but instead use a more descriptive message for users who don't know about the intrinsic details of the cli code.
I'd suggest something like: "Error executing GET endpoint from edge applications API"

(The same applies for other similar cases below)